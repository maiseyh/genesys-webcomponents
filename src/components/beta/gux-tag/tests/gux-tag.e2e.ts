import { newE2EPage } from '@stencil/core/testing';

describe('gux-radial-progress', () => {
  [
    '<gux-tag-beta lang="en">default</gux-tag-beta>',
    '<gux-tag-beta lang="en" color="default">default (explicit)</gux-tag-beta>',
    '<gux-tag-beta lang="en" color="navy">navy</gux-tag-beta>',
    '<gux-tag-beta lang="en" icon="bolt" color="navy">navy</gux-tag-beta>',
    '<gux-tag-beta lang="en" icon="bolt" color="navy" value="3" removable>navy</gux-tag-beta>'
  ].forEach((html, index) => {
    it(`should display component as expected (${index + 1})`, async () => {
      const page = await newE2EPage({ html });
      const element = await page.find('gux-tag-beta');

      expect(element.outerHTML).toMatchSnapshot();
    });
  });
});
