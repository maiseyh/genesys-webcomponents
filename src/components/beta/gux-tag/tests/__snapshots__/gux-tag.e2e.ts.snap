// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gux-radial-progress should display component as expected (1) 1`] = `"<gux-tag-beta lang=\\"en\\" class=\\"hydrated\\"><!----><div class=\\"gux-tag gux-default\\"><div class=\\"gux-tag-text\\">default</div></div></gux-tag-beta>"`;

exports[`gux-radial-progress should display component as expected (2) 1`] = `"<gux-tag-beta lang=\\"en\\" color=\\"default\\" class=\\"hydrated\\"><!----><div class=\\"gux-tag gux-default\\"><div class=\\"gux-tag-text\\">default (explicit)</div></div></gux-tag-beta>"`;

exports[`gux-radial-progress should display component as expected (3) 1`] = `"<gux-tag-beta lang=\\"en\\" color=\\"navy\\" class=\\"hydrated\\"><!----><div class=\\"gux-tag gux-navy\\"><div class=\\"gux-tag-text\\">navy</div></div></gux-tag-beta>"`;

exports[`gux-radial-progress should display component as expected (4) 1`] = `"<gux-tag-beta lang=\\"en\\" icon=\\"bolt\\" color=\\"navy\\" class=\\"hydrated\\"><!----><div class=\\"gux-tag gux-navy\\"><div class=\\"gux-tag-icon-container\\"><gux-icon class=\\"gux-tag-icon hydrated\\" icon-name=\\"bolt\\"><div class=\\"gux-icon-container\\"><svg xmlns=\\"http://www.w3.org/2000/svg\\" data-name=\\"Layer 1\\" viewBox=\\"0 0 32 32\\" aria-hidden=\\"true\\"><path d=\\"M23.88 8.9a.68.68 0 01.13.79l-9.74 20.86a.8.8 0 01-.76.45h-.25a.76.76 0 01-.46-.34.67.67 0 01-.08-.54l3.55-14.57L9 17.34h-.22a.83.83 0 01-.56-.2.66.66 0 01-.23-.7l3.57-14.86a.75.75 0 01.29-.42.81.81 0 01.5-.16h5.91a.8.8 0 01.58.23.71.71 0 01.24.53.72.72 0 01-.09.33l-3.08 8.35L23 8.66h.22a.86.86 0 01.66.24z\\"></path></svg></div></gux-icon></div><div class=\\"gux-tag-text\\">navy</div></div></gux-tag-beta>"`;

exports[`gux-radial-progress should display component as expected (5) 1`] = `"<gux-tag-beta lang=\\"en\\" icon=\\"bolt\\" color=\\"navy\\" value=\\"3\\" removable=\\"\\" class=\\"hydrated\\"><!----><div class=\\"gux-tag gux-navy\\"><div class=\\"gux-tag-icon-container\\"><gux-icon class=\\"gux-tag-icon hydrated\\" icon-name=\\"bolt\\"><div class=\\"gux-icon-container\\"><svg xmlns=\\"http://www.w3.org/2000/svg\\" data-name=\\"Layer 1\\" viewBox=\\"0 0 32 32\\" aria-hidden=\\"true\\"><path d=\\"M23.88 8.9a.68.68 0 01.13.79l-9.74 20.86a.8.8 0 01-.76.45h-.25a.76.76 0 01-.46-.34.67.67 0 01-.08-.54l3.55-14.57L9 17.34h-.22a.83.83 0 01-.56-.2.66.66 0 01-.23-.7l3.57-14.86a.75.75 0 01.29-.42.81.81 0 01.5-.16h5.91a.8.8 0 01.58.23.71.71 0 01.24.53.72.72 0 01-.09.33l-3.08 8.35L23 8.66h.22a.86.86 0 01.66.24z\\"></path></svg></div></gux-icon></div><div class=\\"gux-tag-text\\">navy</div><button class=\\"gux-tag-remove-button\\" type=\\"button\\"><gux-icon class=\\"gux-tag-remove-icon hydrated\\" icon-name=\\"ic-close\\" screenreader-text=\\"Remove tag\\"><div class=\\"gux-icon-container\\"><svg xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 16 16\\" aria-label=\\"Remove tag\\"><path d=\\"M12.95 4.465L9.414 8l3.535 3.535-1.414 1.415L8 9.415 4.465 12.95l-1.414-1.415L6.586 8 3.051 4.465l1.414-1.414L8 6.586l3.535-3.535 1.415 1.414z\\"></path></svg></div></gux-icon></button></div></gux-tag-beta>"`;
